.github/workflows/ci-spring-boot-build-publish-image.yml

```
@@ -197,7 +197,7 @@ jobs:
       - name: Upload trivy report as a Github artifact
         uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808 # pin@v4.3.3
         with:
-          name: trivy-sbom-report
+          name: "sbom-${{ steps.sbom-name.outputs.sbom-artifact-id }}-${{ steps.set-image-tag.outputs.image-tag }}.json"
```

Var det meningen å ha med **.json** der?

```suggestion
          name: "sbom-${{ steps.sbom-name.outputs.sbom-artifact-id }}-${{ steps.set-image-tag.outputs.image-tag }}"
```